src/server/getters/get-reporting-fees.ts

```diff
@@ -357,7 +364,11 @@ function getParticipantEthFees(db: Knex, augur: Augur, reporter: Address, univer
     db.raw("IFNULL(cashParticipant.balance,0) as cashParticipant"),
     db.raw("IFNULL(participationTokenSupply.supply,0) as participationTokenSupply"),
     "disavowed"]).from("all_participants");
-  participantQuery.leftJoin("balances as feeToken", "feeToken.owner", "all_participants.participantAddress");
+  participantQuery.leftJoin("balances as feeToken", function () {
+    this
+      .on("feeToken.owner", db.raw("all_participants.participantAddress"))
+      .andOn("feeToken.token", "!=", db.raw("?", augur.contracts.addresses[augur.rpc.getNetworkID()].Cash));
```

Comment:
Right! or we double count the cash as fees!! 😬

Reply:
🤔 what if someone transferred shares to a participant?

Reply:
just like, market shares? You're creating a balance that isn't Cash

Reply:
So one of the recent changes i made, is that FeeTokens and participation tokens have rows in tokens table, with symbols of ParticipationToken and FeeToken@${feeWindowAddress} perhaps that would be a more accurate filter?

Reply:
https://app.clubhouse.io/augur/story/10938/fix-augur-node-getreportingfees-to-protect-against-random-shares-showing-up-on-a-reporting-participant-doubling-unclaimed

Reply:
Discussed this in discord. Fixing will be a bigger issue we'll address later